nengo/utils/compat.py

```diff
@@ -30,6 +32,13 @@ def ensure_bytes(s):
         assert isinstance(s, bytes)
         return s
 
+    def replace(src, dst):
```

Comment:
Why not use `shutil.move`? It's part of the standard library and does something similar but perhaps more robust...